libs/utils/utils.js

```diff
@@ -198,6 +206,34 @@ export function getLocale(locales, pathname = window.location.pathname) {
   return locale;
 }
 
+export function getLanguage(languages, locales, pathname = window.location.pathname) {
+  const defaultLang = { ietf: 'en', tk: 'hah7vzn.css', prefix: '/en' };
+  if (!languages) {
+    return defaultLang;
+  }
+  const split = pathname.split('/');
+  const baseSplit = [LANGSTORE, PREVIEW].includes(split[1]) ? 1 : 0;
+  const languageString = split[baseSplit + 1];
+  const region = split[baseSplit + 2];
```

Comment:
If this was called `const region = urlPath[locOffset +2]`  that'd be a lot more clear what it means. Reading `split[baseSplit+2]` is not intuitive at all.

As a good mind-model & pattern, reading any single line of code in isolation should still provide you with most of the context that you need to understand what it does.